src/Mod/PartDesign/App/FeatureSketchBased.cpp

```diff
@@ -558,7 +570,7 @@ void ProfileBased::generatePrism(TopoDS_Shape& prism,
         if (method == "ThroughAll")
             // "ThroughAll" is modelled as a very long, but finite prism to avoid problems with pockets

```

Comment:
also here should be an updated comment what s done and the comment info that 1E6 once created problems is quite meaningless and should go.

Reply:
I removed the comment. I think the name of the function explains it well enough.